policyengine_us/parameters/gov/states/hi/tax/income/alternative_tax/income_threshold.yaml

```diff
@@ -0,0 +1,25 @@
+description: Hawaii qualifies filers for the alternative tax on capital gains with taxable income at or above this amount, based on filing status.
+metadata:
+  label: Hawaii alternative tax on capital gains income threshold
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: Hawaii Resident Income Tax Instructions N-11 Rev.2022 
+    href: https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=33
+  - title: Hawaii Resident Income Tax Instructions N-11 Rev.2021 
+    href: https://files.hawaii.gov/tax/forms/2021/n11ins.pdf#page=33
+  - title: Hawaii Income Tax Law 235-71.5 Alternative tax for corporations
```

Comment:
Wrong source right? This is for corporations and doesn't support the parameter values